gallery/src/main.scss

```diff
@@ -110,3 +110,28 @@
     display: none;
   }
 }
+
+// Accessibility Checklist Page
+.nx-list.nx-list--no-bullets {
+  .nx-list__item {
+    list-style: none;
+  }
+}
+
+.nx-tile--accessibility-list {
+  .nx-radio-checkbox__content {
+    overflow: unset;
+    text-overflow: unset;
+    white-space: normal;
+  }
+
+  .nx-text-link {
+    margin-left: 24px;
```

Comment:
```suggestion
    margin-left: var(--nx-spacing-6x);
```

Reply:
Actually this isn't doing anything.  It's being overridden by `.nx-list__actions > :first-child`